src/hardware/cmos.cpp

```diff
@@ -63,12 +63,12 @@ static void cmos_timerevent(uint32_t /*val*/)
 static void cmos_checktimer(void) {
 	PIC_RemoveEvents(cmos_timerevent);
 	if (cmos.timer.div<=2) cmos.timer.div+=7;
-	cmos.timer.delay=(1000.0f/(32768.0f / (1 << (cmos.timer.div - 1))));
+	cmos.timer.delay = (1000.0 / (32768.0 / (1 << (cmos.timer.div - 1))));
```

Comment:
We could put the `1000.0 / (32768.0 ..` portion in a constexpr.

The problem I'm faced with is I don't know what a tenth of 32k is supposed to represent, and the original developer preferred to show the numerator and divisor instead of hard-coding some shorter representation.

So I'm hands-off, unless I can add value.